plugins/woocommerce/client/legacy/js/admin/woocommerce_admin.js

```diff
@@ -629,6 +629,20 @@
 			return buttonText;
 		}
 
+		const generatingContentPhrases = [
+			woocommerce_admin.i18n_product_description_gpt_generating_content_1,
+			woocommerce_admin.i18n_product_description_gpt_generating_content_2,
+			woocommerce_admin.i18n_product_description_gpt_generating_content_3,
+		];
+
+		let generatingContentPhraseInterval = null;
+
+		function getGeneratingContentPhrase() {
+			return generatingContentPhrases[
+				Math.floor( Math.random() * generatingContentPhrases.length )
```

Comment:
I'm getting this error when clicking "Write description"
![image](https://user-images.githubusercontent.com/13437655/232778330-314f8706-7d9d-45ba-accf-78f7e9e4e87d.png)
It might be related to this. Maybe you should subtract 1 from here?

Reply:
@nathanss Oops!! It would help if I included file with the additional messages 😳 added in e122ece

Reply:
Note: This code for getting a random array element is correct, since `Math.random` returns a number between 0 and less than 1. It never is 1.